operator/v1/types_console.go

```diff
@@ -202,6 +205,49 @@ type AddPage struct {
 	DisabledActions []string `json:"disabledActions,omitempty"`
 }
 
+type PerspectiveState string
+
+// These are valid perspective states. "Enabled" means the perspective is shown.
+// "Disabled" means the Perspective is hidden.
+// "AccessReview" means access review check is required to show or hide a Perspective.
+const (
+	Enabled      PerspectiveState = "Enabled"
+	Disabled     PerspectiveState = "Disabled"
+	AccessReview PerspectiveState = "AccessReview"
+)
+
+type PerspectiveAccessReview struct {
+	// requiresAccessReview defines a list of permission checks. The perspective will only be shown when all checks are successful.
+	// +optional
+	RequiresAccessReview []authorizationv1.ResourceAttributes `json:"requiresAccessReview,omitempty"`
```

Comment:
Are there fields present within ResourceAttributes that you do not intend to support straight away?